src/framework/preferences/js/Panels.js

```diff
@@ -683,20 +706,33 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
                 }
             }
         },
-        protoTree: {
-            label: {messagekey: "textSizeLabel"},
-            smallIcon: {messagekey: "textSizeSmallIcon"},
-            largeIcon: {messagekey: "textSizeLargeIcon"},
-            multiplier: {messagekey: "multiplier"},
-            textSizeDescr: {messagekey: "textSizeDescr"}
-        },
         sliderOptions: {
             orientation: "horizontal",
             step: 0.1,
             range: "min"
         }
     });
 
+    fluid.defaults("fluid.prefs.textSize.textfieldButtons", {
```

Comment:
I'm not familiar with how or how well the text to speech tool works with the touch screen on mobile devices. It certainly concerns me when there is no any label or description being set in this button component. No matter how well the text to speech works on mobile devices, nothing would be announced when these buttons gain focus. Are there any label/description design for UIO mobile, @jhung? Or my concern doesn't make sense with mobile devices?